docs/fastdds/xml_configuration/common.rst

```diff
@@ -372,6 +375,17 @@ LatencyBudget
 | ``<duration>``            | See :ref:`latencybudgetqospolicy`.              | :ref:`DurationType` | 0                |
 +---------------------------+-------------------------------------------------+---------------------+------------------+
 
+.. _xml_disableheartbeatpiggyback:
+
+DisableHeartbeatPiggyback
+"""""""""""""""""""""""""
+
++------+---------------------------------------+--------------+-----------+
+| Name | Description                           | Values       | Default   |
++======+=======================================+==============+===========+
+|      | See :ref:`disableheartbeatpiggyback`. | ``BoolType`` | ``false`` |
```

Comment:
```suggestion
|      | See :ref:`disableheartbeatpiggyback`. | ``bool`` | ``false`` |
```

Even though it is a XML `BoolType`, other policies just state the type is `bool`.